src/DataStructures/Python/Matrix.cpp

```diff
@@ -19,20 +19,19 @@ namespace py_bindings {
 void bind_matrix() {
   // Wrapper for basic Matrix operations
   bp::class_<Matrix>("Matrix", bp::init<size_t, size_t>())
-      .def("size",
+      .def("shape",
```

Comment:
make this an attribute please (`add_property` in boost)

Reply:
This is pretty difficult to do, since shape is not a member of the Dynamic Matrix class, and there doesn't seem to be any shape function for the Blaze Matrices.  I could make separate attributes for rows and columns but otherwise I don't know how to make this feasible.

Reply:
I don't know how to do this either. Given that this class is temporary for getting the rest of the IO parts wrapped I'd prefer not to hold up on this. Would that be okay with you @nilsleiffischer? The long term plan is to figure out how to turn `Matrix` into a numpy array anyway.

Reply:
It should be a simple matter of passing `add_property` a getter function, see https://www.boost.org/doc/libs/1_61_0/libs/python/doc/html/tutorial/tutorial/exposing.html#tutorial.exposing.class_properties
If this turns out too complicated, we can postpone, but it appears quite simple on first glance.

Reply:
In fact (still on first glance) it looks to me this would work by just replacing `def` with `add_property`.

Reply:
Yeah, sorry, I guess that does work,  thanks, I didn't realize the getter function could take an argument.